test/e2e/e2e.go

```diff
@@ -188,6 +218,17 @@ func CreateServiceAccountAndBinding(clients *test.Clients, name string, logger *
 	return nil
 }
 
+// CreateService will create a Service
```

Comment:
I think rather than create a service, using the route stuff will handle the istio related stuff. But, takes a depedency on serving, which I think we already might have?

Reply:
we do at the moment but as more things move out we might not in the future?